frontend/package.json

```diff
@@ -23,7 +23,8 @@
     "start:production": "make start-production",
     "start:webnode": "make start-webnode",
     "tests": "make test",
-    "tests:headless": "make test-headless"
+    "tests:headless": "make test-headless",
+    "vendor:styles": "cd vendor/openmina-styles && npm run update:styles"
```

Comment:
Is this a new command to run when updating the styles?

Reply:
exactly! When a core change is done on the styles (our internal style library), we run this to generate the style file (one big file) and move it in the dashboard styles location so that its reflected in the view